vue-frontend/src/components/contribution/DesktopLanding.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <section class="tw-mb-32 lg:tw-mb-48 tw-mt-[25px]">
+    <div
+      class="tw-relative xll:tw-container tw-w-full tw-h-full tw-bg-gradient-to-r tw-from-[#FFFFFF] tw-via-[#F2709C]/[0.05] tw-to-[#FF9472]/[0.05] tw-bg-gradient-[180.05deg] tw-pb-16 lg:tw-pt-[3rem]"
+    >
```

Comment:
Make the background color two side lights, set the bottom color the same way you already set the top color.